src/change_stream.ts

```diff
@@ -94,8 +99,8 @@ export interface PipeOptions {
  * @public
  */
 export interface ChangeStreamOptions extends AggregateOptions {
-  /** Allowed values: ‘default’, ‘updateLookup’. When set to ‘updateLookup’, the change stream will include both a delta describing the changes to the document, as well as a copy of the entire document that was changed from some time after the change occurred. */
-  fullDocument?: string;
+  /** Allowed values: ‘updateLookup’. When set to ‘updateLookup’, the change stream will include both a delta describing the changes to the document, as well as a copy of the entire document that was changed from some time after the change occurred. */
+  fullDocument?: 'updateLookup';
```

Comment:
I think we may have to keep this as string type to avoid breakage. Maybe let's add a type test and a TODO so we can change this in v5?